### ISSUE TITLE
Deoplete backend

### ISSUE THREAD

**Issue by expipiplus1** [myfreeweb/intero.nvim#4]
A deoplete backend would be great. It's often more pleasant then working via omnifunc.

**Comment by valpackett** [comment]
How is that more pleasant? Shouldn't they display the exact same interface?

intero.nvim is so fast it doesn't need to be used asynchronously.

**Comment by expipiplus1** [comment]
Deoplete allows quite a lot of configuration. 

Perhaps more importantly, it allows displaying completions from various sources at the same time, so one wouldn't be just limited to intero's suggestions.

**Comment by valpackett** [comment]
Doesn't deoplete have an omnifunc source?

**Comment by expipiplus1** [comment]
yes, I'm not sure what the state of it is. Last time I used it it wasn't really usable: slow and lacking some functionality.

**Comment by rgrinberg** [comment]
Deoplete is quite a bit nicer as it does more than just prefix completion.
